### DISCUSSION TITLE
with ChatGPT Plus, is it possible to share the cookie for many users

### DISCUSSION THREAD

**Discussion by danielaki** [acheong08/ChatGPT#939] [General]
With ChatGPT Plus, is it possible to share the cookie for many users?

**Comment by sparklesx2390** [comment]
Yes, it is possible! Currently, there is a working cookie somewhere that I tried personally from a Bangladesh website. However, it is important to remember that this comes with a downside; if you consent to releasing your ChatGPT Plus cookie to the public, the personal demand for that cookie (account) will be very high, mostly because of the uprising trend of GPT-4 and Plugins, so make sure to be aware of that.
